diff --git a/dist/admin/pages/EditView/components/FormInputs/Relations/Relations.js b/dist/admin/pages/EditView/components/FormInputs/Relations/Relations.js
index bb267e1a583c8917c643a0c7e0d58830460bf2f9..907d7f2af1360f63be26e054bc9149bcd8d48233 100644
--- a/dist/admin/pages/EditView/components/FormInputs/Relations/Relations.js
+++ b/dist/admin/pages/EditView/components/FormInputs/Relations/Relations.js
@@ -85,7 +85,7 @@ var React__namespace = /*#__PURE__*/_interopNamespace(React);
 }
 /* -------------------------------------------------------------------------------------------------
  * RelationsField
- * -----------------------------------------------------------------------------------------------*/ const RELATIONS_TO_DISPLAY = 5;
+ * -----------------------------------------------------------------------------------------------*/ const RELATIONS_TO_DISPLAY = 50;
 const ONE_WAY_RELATIONS = [
     'oneWay',
     'oneToOne',
diff --git a/dist/admin/pages/EditView/components/FormInputs/Relations/Relations.mjs b/dist/admin/pages/EditView/components/FormInputs/Relations/Relations.mjs
index 4d908a8daea90be78e533f479ea68a35b1c839ce..ef16e5254bb351e1a642b84b5ff711b700da84f1 100644
--- a/dist/admin/pages/EditView/components/FormInputs/Relations/Relations.mjs
+++ b/dist/admin/pages/EditView/components/FormInputs/Relations/Relations.mjs
@@ -63,7 +63,7 @@ import { getCollectionType, RelationModalRenderer } from './RelationModal.mjs';
 }
 /* -------------------------------------------------------------------------------------------------
  * RelationsField
- * -----------------------------------------------------------------------------------------------*/ const RELATIONS_TO_DISPLAY = 5;
+ * -----------------------------------------------------------------------------------------------*/ const RELATIONS_TO_DISPLAY = 50;
 const ONE_WAY_RELATIONS = [
     'oneWay',
     'oneToOne',
